panel/src/components/Views/SystemView.vue

```diff
@@ -106,7 +103,18 @@ export default {
 
 		await promiseAll(promises);
 
-		console.info("System health checks ended.");
+		console.info(
+			`System health checks ended. ${
+				promises.length - this.accessible.length
+			} issues found.`
+		);
+
+		// print exceptions from the update check to console for debugging
+		if (this.exceptions.length > 0) {
+			console.info("----");
+			console.info("Running the update check for Kirby and plugins.");
```

Comment:
Given that the output is only logged when there are exceptions, maybe we should even reword the message to something like "The following errors occurred during the update check of Kirby and/or plugins:". Or alternatively the message should be printed always and then there should also be a final "Update check ended." message.